
Diagnostics:
error: cannot move local `x` which is still borrowed
   ┌─ tests/reference-safety/v1-tests/call_transfer_borrows.move:17:9
   │
13 │         let x_ref = &x;
   │                     -- previous local borrow
14 │         let y_ref = &mut y;
15 │         let r = take_imm_mut_give_mut(x_ref, y_ref);
   │                 ----------------------------------- used by mutable call result
16 │         *x_ref;
17 │         move x; // error in v2 (bug in v1)?
   │         ^^^^^^ moved here
18 │         *r = 1;
   │         ------ conflicting reference `r` used here

error: mutable reference in local `y_ref` requires exclusive access but is borrowed
   ┌─ tests/reference-safety/v1-tests/call_transfer_borrows.move:29:9
   │
26 │         let r = take_imm_mut_give_imm(x_ref, y_ref);
   │                 ----------------------------------- previous call result
   ·
29 │         *y_ref;
   │         ^^^^^^ requirement enforced here
30 │         *x_ref;
31 │         *r;
   │         -- conflicting reference `r` used here
